changelogs/fragments/ssh-255-and-python-m.yaml

```diff
@@ -0,0 +1,8 @@
+---
+bugfixes:
+- If a module exited with error 255 When using the ssh connection plugin, the
+  connection plugin would swallow the error and say we had an ssh error instead.
+  The new code fixes this for select PHP errors and Python tracebacks.
```

Comment:
PHP? PHP as a PHP language? 0_O
Where does that come from?

Reply:
Modules can be any language